### DISCUSSION TITLE
How to know the IP assigned from AP to the new wifi without serial monitor?

### DISCUSSION THREAD

**Discussion by Bluesign71** [Hieromon/AutoConnect#445] [How can I?] [answered]
Hallo! I try to explain what I mean:
1) I access to esp32 by captive portal in AP mode and set the wifi credential, and everythings works fine
2) Once connecting to the wifi in station mode, the captive portal freeze (and that's ok) but how could I know the ip address assigned to my esp32 in the lan ??? I can't connect to a serial monitor from the IDE and just have access to my phone. 

So I don't know how to reach the autoconnect webserver when operating in station mode.

Could You please help me? thanks.....

**Comment by Hieromon** [comment, marked as answer]
You can choose several ways to find the IP address of ESP32.
1. Use your WiFi router some utility to list the DHCP leases. Consult your router’s manual to check how to access it.
2. Use Network scanner Apps on your cellphone. (e.g. Net Analyzer for iOS, Network Scanner for Android)
3. Launch the [mDNS](https://hieromon.github.io/AutoConnect/adnetwork.html#combination-with-mdns) with ESP32, you can use the mDNS explorer of your client device to search for it. However, from Windows 10, it seems that it may not be detected well even if the firewall is made transparent.
4. Assign a static IP to the ESP32 with [AutoConnectConfig](https://hieromon.github.io/AutoConnect/adnetwork.html#static-ip-assignment-as-a-client).
